### PR TITLE
[BUG] Add Location to college view and fix spacing issue with scholarship

### PR DESCRIPTION
**Pull request by MikolJiron**

### CODE REVIEW

File: src/main/webapp/js/finscholar/collegepageview/collegepageview.js [hunk 1]
```diff
@@ -43,13 +43,16 @@ class CollegePageView extends SinglePageView {
     }
   }
  
-  /** @private Rander buttons for each scholarship related to current college. */
+  /** @private Render buttons for each scholarship related to current college. */
```
Comment (Lorraine-Lyu) [inline src/main/webapp/js/finscholar/collegepageview/collegepageview.js:46]: Thanks.

**Review by Lorraine-Lyu — APPROVED** [review]

**Review by tinagrasso — CHANGES_REQUESTED** [review]
screenshot?

**Review by hzimz — APPROVED** [review]

**Review by CarlosM10011 — APPROVED** [review]

**Comment by MikolJiron** [comment]
Screenshot 2020-08-05 at 10 54 35 AM - Display 2

**Review by tinagrasso — CHANGES_REQUESTED** [review]
location isn't aligned with the rest of the text.

**Comment by MikolJiron** [comment]
Screenshot 2020-08-05 at 12 46 49 PM

**Comment by MikolJiron** [comment]
Fixed!

**Review by tinagrasso — APPROVED** [review]

**Pull request merged**